lib/mempool/mempool.js

```diff
@@ -282,6 +282,27 @@ class Mempool extends EventEmitter {
         this.untrackClaim(entry);
     }
 
+    // Remove all GooSig based airdrops from the mempool
+    // when the block at the height that disables GooSig
+    // is added to the mempool. It would be wasteful to do
+    // this check on every block after GooSig is disabled,
+    // so only do it once on the block height that disables
+    // GooSig.
+    if (this.network.airdrop.disableGoosig) {
+      if (block.height === this.network.airdrop.disableGoosigHeight) {
+        for (const [hash, entry] of this.airdrops.values()) {
```

Comment:
By specifying `.values()` here you will only get the entries back (not the hashes) and the `for` loop will throw as non-iterable

Reply:
Good catch, there is test coverage for this codepath now.